OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -71,26 +71,13 @@ public IReadOnlyPackage OpenPackage(string filename)
 		public IReadOnlyPackage OpenPackage(string filename, IReadOnlyPackage parent)
 		{
 			// HACK: limit support to zip and folder until we generalize the PackageLoader support
-			if (parent is Folder)
+			if (filename.EndsWith(".zip", StringComparison.InvariantCultureIgnoreCase) ||
+				filename.EndsWith(".oramap", StringComparison.InvariantCultureIgnoreCase))
 			{
```

Comment:
I kept them for readability, otherwise the `using` runs up against the second line of the conditional at the same indentation level.